frontend/packages/dev-console/integration-tests/support/pages/app.ts

```diff
@@ -0,0 +1,202 @@
+import { devNavigationMenu, switchPerspective } from '../constants/global';
+
+export const app = {
+  waitForLoad: (timeout: number = 30000) => {
+    cy.get('.co-m-loader', { timeout }).should('not.be.visible');
+  },
+  login: () => {
+    cy.get('body').then(($body) => {
+      if ($body.find('a[title="Log in with kube:admin"]').length) {
+        cy.get('a[title="Log in with kube:admin"]').click();
+        cy.url().should('include', 'login');
+      }
+    });
+    cy.get('#inputUsername').type(Cypress.env('username'));
+    cy.get('#inputPassword').type(Cypress.env('password'));
+    cy.get('[type="submit"]').click();
+    cy.get('[aria-label="Help menu"]').should('be.visible');
+  },
+};
+
+export const perspective = {
+  verifyPerspective: (perspectiveName: string) => {
+    cy.byLegacyTestID('perspective-switcher-toggle').should('contain.text', perspectiveName);
+  },
+
+  switchTo: (perspectiveName: switchPerspective) => {
+    cy.byLegacyTestID('perspective-switcher-toggle').click();
+    switch (perspectiveName) {
+      case switchPerspective.Administrator: {
+        cy.get('li[role="menuitem"]')
+          .contains('Administrator')
+          .click();
+        cy.get('body div').then(($el) => {
+          if ($el.find('#guided-tour-modal').length !== 0) {
```

Comment:
This shouldn't be part of a `switchTo` assertion.  Other tests migh switch persepective and be testing for the guided tour modal!  Perhaps an additional param `closeGuidedTour: true`. -just so other devs know what the switch is/can do.